lib/internal/events/symbols.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const {
+  Symbol
```

Comment:
Can you rename this file to `lib/internal/events/utils.js` please?

```suggestion
  Symbol,
```